volatility3/cli/text_renderer.py

```diff
@@ -345,7 +345,7 @@ def get_render_options(self) -> List[interfaces.renderers.RenderOption]:
 
     def output_result(self, outfd, result):
         """Outputs the JSON data to a file in a particular format"""
-        outfd.write(json.dumps(result, indent = 2, sort_keys = True))
+        outfd.write("{}\n".format(json.dumps(result, indent = 2, sort_keys = True)))
```

Comment:
We do now allow format strings (so `outfd.write(f"{json.dumps(result, indent = 2, sort_keys = True)}\n")` or even just `+ "\n"` but this is fine...  5;)

Reply:
Thank you for the good information! 🙌